install_config/aggregate_logging_sizing.adoc

```diff
@@ -162,6 +156,44 @@ size].
 
 xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[Learn more about installing EFK].
 
+
+[[install-config-aggregate-logging-sizing-guidelines-large-cluster-installation]]
+=== For large clusters
+
+At 100 nodes or more, it is recommended to pre-pull the logging images first and to set `ImagePullPolicy: IfNotPresent` in the logging-deployer.yaml file. After deploying the logging infrastructure pods (elasticsearch, kibana and curator), node labeling should be done in steps of 20 nodes at a time. For example:
+
+Using a simple loop:
+
+----
+$ while read node; do oc label nodes $node logging-infra-fluentd=true; done < 20_fluentd.lst
+----
+
+The below also works:
+
+----
+$ oc label nodes 10.10.0.{100..119} logging-infra-fluentd=true
+----
+
+This is done in order to avoid a possible thundering herd scenario against your image registry.
```

Comment:
Labeling nodes in groups paces the DaemonSet used by OpenShift logging, helping to avoid contention on shared resources such as the image registry.